src/Features/Core/Portable/ChangeAccessibilityModifier/AbstractChangeAccessibilityModifierCodeFixProvider.cs

```diff
@@ -0,0 +1,116 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#nullable disable
+
+using System;
+using System.Collections.Immutable;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CodeActions;
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.Editing;
+using Microsoft.CodeAnalysis.LanguageServices;
+using Microsoft.CodeAnalysis.Shared.Extensions;
+
+namespace Microsoft.CodeAnalysis.ChangeAccessibilityModifier
+{
+    internal abstract class AbstractChangeAccessibilityModifierCodeFixProvider : CodeFixProvider
+    {
+        private readonly string _title = FeaturesResources.Change_accessibility;
+        private readonly string _titleFormat = FeaturesResources.Change_accessibility_to_0;
```

Comment:
Can you show what this looks like?  It looks like it will look similar to 

```
Change accessibility
   Change accessibility to 'private protected'
```

Which woudl be pretty verbose.  MOre ideal would be:

```
Change accessibility to:
   private protected
```

with 'isInlinable = false' so that this always doesn't clutter hte main list and always becomes a suboption of the main option.